docs/docs/reference/configuration.md

```diff
@@ -7,9 +7,6 @@ title: Configuration
 OPEN AN ISSUE IF YOU WOULD LIKE TO MAKE ADJUSTMENTS HERE AND MAINTAINERS WILL HELP YOU LOCATE THE RIGHT
 FILE -->
 
-If file `$HOME/.oathkeeper.yaml` exists, it will be used as a configuration file
-which supports all configuration settings listed below.
-
```

Comment:
Looks like this was removed from running `npm run gen:config`

Reply:
Possible, we probably did not run it yet since the script was updated at some point.